### ISSUE TITLE
Push notification only works for opened app?

### ISSUE THREAD

**Issue by cuiying408** [RocketChat/Rocket.Chat.iOS#2558]
- Your Rocket.Chat app version: 3.2.2
- Your Rocket.Chat server version: 0.71.1
- Device (or Simulator) you're running with: iPhone 7

I can see push notification while app is opened with user list. But I can't get push notification after tapping home button and while app is in background mode.

**Comment by rafaelks** [comment]
@cuiying408 Do you have push notifications enabled for your app? You can check that in Settings -> Notifications. Also, are you using our push gateway to send the notification?

**Comment by cuiying408** [comment]
Yes, it's enabled. And I am using my own gateway to send the notification.

**Comment by rafaelks** [comment]
@cuiying408 If you're using your own gateway, are you using also your own fork of the iOS app?

**Comment by cuiying408** [comment]
I've set APN key and APN cert in admin panel.

**Comment by rtizi** [comment]
@cuiying408 Hi
Please tell me in which fields you entered the data and where it came from, how to get the key from the certificate from the Apple site?

**Comment by BalajiSanjeevi** [comment]
@cuiying408 
Are you still not getting the push notification when the app is in background?
